src/examples/go_monorepo_workflows.yml

```diff
@@ -0,0 +1,187 @@
+description: |
+  This example shows:
+   - Specific version of the isopod.
+   - this is for go application
+   - custom quality checks
+usage:
+  version: 2.1
+
+  orbs:
+    go: circleci/go@1.7.1
+    ric-orb: ricardo/ric-orb@dev:e4c7a02
```

Comment:
dont forget to swtich back to `ricardo/ric-orb@2` once you released your orb changes